docs/build_version_doc/update_all_version.sh

```diff
@@ -93,7 +93,7 @@ function update_mxnet_css {
 for tag in $tag_list; do
     # This Python script is expecting the tag_list.txt and it will use that as the entries to populate
 
-    python AddVersion.py --root_url "$root_url" --file_path "$built/versions/$tag" --current_version "$tag" || exit 1
+    python AddVersion.py --root_url "$root_url" --file_path "$built/versions/$tag" --current_version "$tag" --default_tag "$tag_default" || exit 1
```

Comment:
Did you need to flip the argument or could you be consistent?
default_tag vs tag_default?

Reply:
Changed the variable in AddVersion.py to be tag_default